taf/auth_repo.py

```diff
@@ -160,6 +171,17 @@ def get_target(self, target_name, commit=None, safely=True):
         else:
             return self.get_json(commit, target_path)
 
+    def get_metadata(
```

Comment:
It's interesting that we didn't already have this. We can probably use it to simplify other parts of out code, but not now